src/main/java/core/basesyntax/figures/Square.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.figures;
+
+import java.util.Locale;
+
+public class Square extends Figure {
+    private double side;
+    private double area;
+
+    public Square(String color, double side) {
+        super(color);
+        this.side = side;
+        this.area = calculateArea();
```

Comment:
Don't calculate and store the area in the constructor. Instead, calculate it on demand in the 'getArea()' method.